modules/core/src/main/java/org/apache/ignite/internal/managers/IgniteMBeansManager.java

```diff
@@ -196,16 +194,14 @@ public void registerAllMBeans(
         registerExecutorMBean("GridSystemExecutor", sysExecSvc);
         registerExecutorMBean("GridClassLoadingExecutor", p2pExecSvc);
         registerExecutorMBean("GridManagementExecutor", mgmtExecSvc);
-        registerExecutorMBean("GridIgfsExecutor", igfsExecSvc);
+        registerExecutorMBean("GridDataStreamExecutor", dataStreamExecSvc);
```

Comment:
Why MBean of this executor changed from stripped to standard?

Reply:
It's not changed. It is still stripped

Reply:
I am wrong. You are right. Now I see the error.